Doc/Prepare-macOS-Big-Sur-HOWTO.md

```diff
@@ -1160,14 +1181,14 @@ Everything in this section must be done **from the biocbuild account**.
 
 ### 4.1 Install Java
 
-Go to https://jdk.java.net/ and follow the link to the latest JDK (JDK
-14 as of April 1, 2020). Then download the tarball for macOS/x64 (e.g.
-`openjdk-14.0.1_osx-x64_bin.tar.gz`) to `~/Downloads/`.
+Go to https://jdk.java.net/ and follow the link to the latest JDK. Then
+download the tarball for your specifc mac (e.g. `openjdk-18.0.1.1_macos-x64_bin.tar.gz`
+or `openjdk-18.0.1.1_macos-aarch64_bin.tar.gz`) to `~/Downloads/`.
```

Comment:
I'd recommend using https://adoptium.net/ instead. There one can easily find which is the latest LTS release. Currently it shows v17 but it will change to v21 in the coming days.
Java 18 is not supported any more.

Reply:
Thanks for the suggestion. I'll update the machines next month. Good to know Java needs an update.